lib/web_ui/lib/pointer.dart

```diff
@@ -154,6 +154,7 @@ class PointerData {
 }
 
 class PointerDataPacket {
-  const PointerDataPacket({this.data = const <PointerData>[]});
+  PointerDataPacket({this.data = const <PointerData>[]});
   final List<PointerData> data;
+  bool handled = true;
```

Comment:
Can this be modelled without mutation? For one, multiple pieces of framework/widget/app code can compete for the value. One can set it to false, then something else comes along and sets it to true. I guess that part can be fixed by making this a method (e.g. `ignore()`) that's either called once or is idempotent.

However, can this be modelled using a `PointerDataPacketResponse` event that's sent back from the framework to the engine? The correlation between `PointerDataPacket` and `PointerDataPacketResponse` can either be established via a unique ID, or by having `PointerDataPacketResponse` wrap the original `PointerDataPacket`.

Reply:
Ah, I left a similar comment on the framework-side PR of this :)

Reply:
The method that I added on the framework is idempotent, once you mark a `PointerDataPacket` as ignored, it can't be rolled back ever (the flip of the boolean happens in a closure).

I've tried to plumb the PointerDataPacketResponse into the framework side of things, and the resulting code is ugly, and very much not backwards compatible.

I'm going to try and simplify this so that the framework can "allowPlatformDefault" or similar on a PointerDataPacket (through the platform dispatcher, I guess), instead of making the DataPacket mutable, see how that looks :)